### PR TITLE
Fix bought from JS to show warning when user presses "no".

### PR DESCRIPTION
**Pull request by MattHolmes123**

### CODE REVIEW

**Comment by MattHolmes123** [comment]
image
When pressing "no" this message is shown straight away now

File: web/static/web/js/pages/bought-from.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 window.addEventListener('load', (event) => {
-  const boughtFrom = document.querySelector('#id_know_bought_from');
-  const boughtFromInfoBox = document.querySelector('#know-bought-from-info-box');
+  "use strict";
```
Comment (mikeylockwood) [inline web/static/web/js/pages/bought-from.js:2]: What's this string for?

File: web/static/web/js/pages/bought-from.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
-  boughtFrom.addEventListener('change', (e) => {
-    const displayStyle = e.target.value == 'false' ? 'block': 'none';
-    boughtFromInfoBox.style.display = displayStyle;
-  });
+  const showBoughtFrom = UTILS.getShowElementFunc("#know-bought-from-info-box")
+  const boughtFromRadios = document.querySelectorAll("input[type=radio][name=know_bought_from]")
 
+  boughtFromRadios.forEach((radio) => radio.addEventListener("change", (e) => {
+    showBoughtFrom(e.target.value === "False")
```
Comment (mikeylockwood) [inline web/static/web/js/pages/bought-from.js:8]: I would `.toLowerCase` this

File: web/static/web/js/pages/bought-from.js (hunk 1 above)
Comment (MattHolmes123) [reply web/static/web/js/pages/bought-from.js:2]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode

File: web/static/web/js/pages/bought-from.js (hunk 2 above)
Comment (MattHolmes123) [reply web/static/web/js/pages/bought-from.js:8]: I mean its the django form value so it will never change but I can do

**Pull request merged**